site/content/blog/2019-10-21-localizing-matterpoll/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Localizing Matterpoll"
+slug: localizing-matterpoll
+date: 2019-11-05T10:49:35+02:00
+categories:
+    - "plugins"
+    - "go"
+author: Ben Schumacher
+github: hanzei
+community: hanzei
+---
+
+[Matterpoll](https://github.com/matterpoll/matterpoll) is a plugin that allows creating polls inside of Mattermost. Given that Mattermost is [localized in 16 different languages](https://docs.mattermost.com/developer/localization.html) we surely want to accomplish the same for Matterpoll. Because we rely on contributors to do the translations we wanted to make sure it's a easy as possible for them to translate new strings and determine if already translated stings needs to be updated because the "source" text changed. On the other hand Matterpoll only has two maintainer ([@kaaka](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no own infrastructure to work with. Using some translation server like [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```

Comment:
```suggestion
[Matterpoll](https://github.com/matterpoll/matterpoll) is a poll plugin within Mattermost. As Mattermost is [localized in 16 different languages](https://docs.mattermost.com/developer/localization.html) we'd like to accomplish the same for Matterpoll. 

Because we rely on contributors to do the translations we wanted to make sure it's as easy as possible for them to translate new strings and determine whether already translated strings should be updated because the source text changed. 

On the other hand Matterpoll only has two maintainers ([@kaaka](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no own infrastructure to work with. Using a translation server such as [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```

Reply:
Could you please elaborate, why you want to break this into different paragraphs?

Reply:
Good question. Shorter paragraphs are easier to read - a long paragraph can be off-putting. I also identified three separate ideas, which I broke up into separate paragraphs. This is also aligned with our style guide that recommends shorter paragraphs (https://docs.mattermost.com/process/sg_grammar-spelling-mechanics.html#paragraphs-and-sentences). 

That said, we can join P2 and P3 and it would align with the style guide and be quite clear.

Reply:
Thanks for the explanation. Will break it into shorter paragraphs :+1: